### PR TITLE
Add VK_EXT_shader_image_atomic_int64 draw VUs

### PR DESCRIPTION
**Pull request by spencer-lunarg**
Makes sure if there is a 64-bit `VkImageView` or 64-bit `OpTypeImage` that both are either 64-bit or 32-bit (only other valid option) 

(related issue https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/2629)

### CODE REVIEW

**Review by arno-lunarg — APPROVED** [review]
LGTM!

File: tests/vklayertests_pipeline_shader.cpp [hunk 1]
```diff
@@ -11508,6 +11508,125 @@ TEST_F(VkLayerTest, ShaderImageAtomicInt64) {
...
+    SetTargetApiVersion(VK_API_VERSION_1_1);
+    AddRequiredExtensions(VK_EXT_SHADER_IMAGE_ATOMIC_INT64_EXTENSION_NAME);
+    ASSERT_NO_FATAL_FAILURE(InitFramework());
+    if (!AreRequiredExtensionsEnabled()) {
+        GTEST_SKIP() << RequiredExtensionsNotSupported() << " not supported";
+    }
+
+    auto image_atomic_int64_features = LvlInitStruct<VkPhysicalDeviceShaderImageAtomicInt64FeaturesEXT>();
+    auto features2 = GetPhysicalDeviceFeatures2(image_atomic_int64_features);
+    if (features2.features.shaderInt64 == VK_FALSE) {
```
Comment (artem-lunarg) [inline tests/vklayertests_pipeline_shader.cpp:11522]: Is it a good idea to skip checking for features if the feature support is guaranteed by the extension?

File: tests/vklayertests_pipeline_shader.cpp [hunk 2]
```diff
@@ -11508,6 +11508,125 @@ TEST_F(VkLayerTest, ShaderImageAtomicInt64) {
     }
 }
 
+TEST_F(VkLayerTest, ShaderImageAtomicInt64Drawtime64) {
```
Comment (artem-lunarg) [inline tests/vklayertests_pipeline_shader.cpp:11511]: typo: ShaderImageAtomicInt32Drawtime64

Comment (spencer-lunarg) [reply tests/vklayertests_pipeline_shader.cpp:11511]: the extension is `VK_EXT_shader_image_atomic_int64` so that is why it is `ShaderImageAtomicInt64`

Comment (artem-lunarg) [reply tests/vklayertests_pipeline_shader.cpp:11511]: Ah, makes sense! I came up with some imaginary encoding...

**Pull request merged**